components/prism-arturo.js

```diff
@@ -0,0 +1,49 @@
+Prism.languages.arturo = {
+    'comment': /;.*/,
+
+    'character': {
+        pattern: /`.`/,
```

Comment:
Please make `comment` and `character` greedy as well. Right now, `';'` is highlighted as a comment.

Reply:
Done 😉